src/index.css

```diff
@@ -1,3 +1,197 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
 * {
-  font-family: sans-serif;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+body {
+  background: rgb(213, 222, 224);
+}
```

Comment:
- [ ] Great job so far. However, there're few things you still need to work on to complete this milestore but trust me you're almost there. It is required to produce the exact design as we have it on the guiding template from [Zeplin](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md). Your background color looks different from the guiding template. Kindly fix that.

![project-rqts](https://user-images.githubusercontent.com/26597432/222463699-36c6178e-3080-4be2-9e6a-c47636e4ec60.png)

- [ ] Also, there's a missing image at the top right corner of your page. Kindly add that and remove text `The book store`.

### Your Design
![your-navbar](https://user-images.githubusercontent.com/26597432/222462915-be748522-ab9f-4ff7-bab3-d62e31f54d54.png)

### Template Design
![temp-navbar](https://user-images.githubusercontent.com/26597432/222462949-561d5de5-e5ed-4e5a-b586-97f54b854679.png)

Reply:
Thank you for the feedback